executor/runtime/docker/docker.go

```diff
@@ -1782,7 +1782,11 @@ func (r *DockerRuntime) setupPostStartLogDirTiniHandleConnection2(parentCtx cont
 
 	if r.dockerCfg.bumpTiniSchedPriority {
 		group.Go(func() error {
-			return setupScheduler(cred)
+			err := setupScheduler(cred)
```

Comment:
Whoa, was this why it didn't work on Mac?

Reply:
Not super sure. `sched_getscheduler` should be able to be run with the unconfined mode. It is also an older `4.19.76-linuxkit` kernel. `CONFIG_RT_GROUP_SCHED=y` on mine.